app/javascript/components/util/DeleteDialog.js

```diff
@@ -78,6 +81,26 @@ class DeleteDialog extends React.Component {
               <div className="character-limit-text">{MAX_REASON_LENGTH - this.state.delete_reason_text.length} characters remaining</div>
             </React.Fragment>
           )}
+          {this.props.type === 'Lab Result' && this.props.promptSymptomOnset && (
+            <div>
+              <p>
+                Since this record does not have a Symptom Onset Date, deleting this positive lab result may result in the record not ever being eligible to
+                appear on the Records Requiring Review line list. Please enter a symptom onset date to prevent this from happening:
+              </p>
+              <Form.Label className="nav-input-label">SYMPTOM ONSET</Form.Label>
+              <DateInput
+                id="symptom_onset_delete_dialog"
+                date={this.state.symptom_onset}
+                minDate={'2020-01-01'}
+                maxDate={moment().add(30, 'days').format('YYYY-MM-DD')}
+                onChange={date => this.setState({ symptom_onset: date })}
+                isClearable={true}
+                placement="bottom"
+                customClass="form-control-lg"
```

Comment:
can you make this `form-control-md` to match the rest of the inputs in this dialog?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/eb3c0c52a98a82e70e2904d5613ab6c85e741543#diff-431caf4d60fa1325346adae1e9dd1b818a55ea030365a593cdf8ebcaaab58dbdR99